CHANGELOG.md

```diff
@@ -14,3 +14,8 @@ Project versions conform to [Semantic Versioning](https://semver.org/)
 - `Removed`: for deprecated features removed in this release
 
 [1.0.0-pre-release]
+
+## 1.1.0
+
+* Fixed styles for headings and the page height for the PDF outputs.
```

Comment:
@geealbers not critical though the `CHANGELOG` entry could be formatted as

```
## [1.1.0]

### Fixed

* CSS styles for headings
* page height for the PDF outputs

```

Reply:
Ok. I'll make that change in the next PR (coming shortly) just so that you don't have to approve this one again. Thanks